ci/infra/vmware/terraform.tfvars.json.ci.example

```diff
@@ -1,6 +1,7 @@
 {
     "vsphere_datastore": "3PAR",
     "vsphere_datacenter": "PROVO",
+    "vsphere_datastore_cluster": null,
```

Comment:
I think deployment is failing because this is set to null. Should be set to `LOCAL-DISKS-CLUSTER` for our current state

Reply:
now that defautls are set to null we can override this for our CI and remove `vsphere_datastore` var from here :D